clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/DcosSpinnakerLbId.groovy

```diff
@@ -0,0 +1,83 @@
+package com.netflix.spinnaker.clouddriver.dcos.deploy.util.id
+
+import static com.google.common.base.Preconditions.checkArgument
+import static com.google.common.base.Strings.nullToEmpty
+
+/**
+ * Represents a hierarchical Spinnaker specific load balancer identifier for DCOS.
+ * Structure - /account/loadBalancerName
+ */
+class DcosSpinnakerLbId {
+    public final static def SAFE_NAME_SEPARATOR = "_"
+
+    private final def marathonPath
+
+    public DcosSpinnakerLbId(final String id) {
+        this.marathonPath = MarathonPathId.parse(nullToEmpty(id)).absolute()
+
+        checkArgument(this.marathonPath.size() == 2, "A part of the DCOS Spinnaker LB ID was missing.")
+    }
+
+    public DcosSpinnakerLbId(final String account, final String loadBalancerName) {
```

Comment:
May want to have this call the constructor above instead of MarathonPathId.parse.  Right now it looks like the two constructors are inconsistent in validation because this one would allow a path with more than 2 parts if `loadBalancerName` contained '/'